cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -337,28 +331,31 @@ enum SurveyCoverageSource {
     CALCULATED = 2;
 }
 
+// Used to specify dimensionality of a seismic object.
 enum Dimensions {
     UNSPECIFIED_DIMENSION = 0;
     TWO_DEE = 2;
     THREE_DEE = 3;
 }
 
+// Enumeration of possible trace header fields used for indexing seismic objects.
 enum TraceHeaderField {
     UNSPECIFIED_TRACE_HEADER_FIELD = 0;
-    ENERGY_SOURCE_POINT = 1;
-    CDP = 2;
-    INLINE = 3;
-    CROSSLINE = 4;
-    SHOTPOINT = 5;
-    CDP_TRACE = 6; // bytes 25-28 of standard trace header: trace number within ensemble
-    OFFSET = 7;  // bytes 37-40 of standard trace header: distance from center of source point to the center of receiver group.
+    ENERGY_SOURCE_POINT = 1; // Valid for 2d seismic objects
+    CDP = 2;                 // Valid for 2d seismic objects
+    INLINE = 3;              // Valid for 3d seismic objects
+    CROSSLINE = 4;           // Valid for 3d seismic objects
+    SHOTPOINT = 5;           // Valid for 2d seismic objects
+    CDP_TRACE = 6;           // Valid for pre-stack seismic objects. Corresponds to bytes 25-28 of standard trace header: trace number within ensemble
+    OFFSET = 7;              // Valid for pre-stack seismic objects. Corresponds to bytes 37-40 of standard trace header: distance from center of source point to the center of receiver group.
```

Comment:
Maybe we ought to have byte offsets in the other fields as well

Reply:
I added byte offsets to the best of my knowledge. Double checking them will be appreciated

Reply:
Looks good!